rust/azure_iot_operations_connector/examples/filemount_azure_device_registry_observation.rs

```diff
@@ -203,7 +203,7 @@ impl FileMountManager {
     fn add_device_endpoint(&self, device_endpoint: &DeviceEndpointRef, asset_names: &[AssetRef]) {
         let file_path = self.dir.as_path().join(device_endpoint.to_string());
         let content: Vec<_> = asset_names.iter().map(|asset| asset.name.clone()).collect();
-        let content = content.join(";");
+        let content = content.join("\n");
```

Comment:
Is there some way we could make this platform independent for safety? I know we're only expecting this to run on a specific platform, but I would feel better if there was some kind of abstraction over the newline for safety.